CHANGELOG.md

```diff
@@ -26,9 +26,18 @@ Release report: TBD
 Wazuh commit: TBD \
 Release report: TBD
 
-### Fixed
+### Added
+
+- Add IT tests FIM registry monitoring using wildcards. ([#4270](https://github.com/wazuh/wazuh-qa/pull/4270)) \- (Framework + Tests)
 - Update schema database version ([#4128](https://github.com/wazuh/wazuh-qa/pull/4128)) \- (Tests)
```

Comment:
This change should be in the ##Changed section

Reply:
Done: https://github.com/wazuh/wazuh-qa/pull/4349/commits/6dfd4473d2755aaf4cfa6a5163ca44431c2789e6